posts/2020-07-30-json-logging-open-liberty-20008.adoc

```diff
@@ -0,0 +1,282 @@
+---
+layout: post
+title: "Access HTTP log fields in JSON logs and write JSON directly to system.out without wrapping on Open Liberty 20.0.0.8"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Access HTTP log fields in JSON logs and write JSON directly to system.out without wrapping on Open Liberty 20.0.0.8 - OpenLiberty.io
+seo-description: With Open Liberty 20.0.0.8 users can now access HTTP log fields in JSON logs and also write JSON directly to system.out without wrapping in the `liberty_message` event.
+blog_description: With Open Liberty 20.0.0.8 users can now access HTTP log fields in JSON logs and also write JSON directly to system.out without wrapping in the `liberty_message` event.
+---
+= Access HTTP log fields in JSON logs and write JSON directly to system.out without wrapping on Open Liberty 20.0.0.8
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+// // // // // // // //
```

Comment:
Can you delete these template comments? We don't need them once you've got the draft in place.

Reply:
Done